oak_functions/loader/src/main.rs

```diff
@@ -190,12 +215,18 @@ async fn main() -> anyhow::Result<()> {
     let wasm_module_bytes = fs::read(&opt.wasm_path)
         .with_context(|| format!("Couldn't read Wasm file {}", &opt.wasm_path))?;
 
-    let lookup_data = LookupData::new_empty(&config.lookup_data_url);
+    let lookup_data = Arc::new(LookupData::new_empty(&config.lookup_data_url));
     if !config.lookup_data_url.is_empty() {
-        lookup_data
```

Comment:
I think that this should either remain here, or the periodic refresh code should first do the refresh before waiting in the loop. At the moment it seems like the lookup data will remain empty for the first period if I am reading the logic correctly.

Reply:
Done, I think it makes more sense now, thanks!